src/AutoRest.CSharp/build/CodeGeneration.targets

```diff
@@ -32,6 +32,8 @@
     <UseDefaultNamespaceAndOutputFolder Condition="'$(UseDefaultNamespaceAndOutputFolder)' == ''">true</UseDefaultNamespaceAndOutputFolder>
     <_AutoRestCommand>node $(AutoRestEntryPoint) --max-memory-size=8192 --skip-csproj --skip-upgrade-check --version=$(AutoRestCoreVersion) $(AutoRestInput) $(AutoRestAdditionalParameters) --use=$(MSBuildThisFileDirectory)../tools/net6.0/any/ --clear-output-folder=true --shared-source-folders=&quot;$(AzureCoreSharedCodeDirectory);$(AutoRestSharedCodeDirectory)&quot;</_AutoRestCommand>
     <_AutoRestCommand Condition="'$(UseDefaultNamespaceAndOutputFolder)' == 'true'">$(_AutoRestCommand) --output-folder=$(MSBuildProjectDirectory)/Generated --namespace=$(RootNamespace)</_AutoRestCommand>
+    <_SaveInputs Condition="'$(SaveInputs)' != ''">-SaveInputs</_SaveInputs>
```

Comment:
```suggestion
    <_SaveInputs Condition="'$(SaveInputs)' == 'true'">-SaveInputs</_SaveInputs>
```

Reply:
Not sure the intent, but I agree with with Crystal that we should consider tightening these up.   If you're trying to catch undefined as well as true, maybe something like:

```xml
<_SaveInputs Condition="'$(SaveInputs)' != 'false'">-SaveInputs</_SaveInputs>
```

Same thought for the other places that Crystal commented.